From 7715d6cd72477af83d95563b69a5f0273bdb719b Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Mon, 11 Jun 2012 20:19:50 +0200
Subject: [PATCH] Fix test env for the stable branch

Need both changes in one commit to pass the gate!

* Nail pep8 dependencies to 1.0.1.

Nails the pep8 deps for tox and test-requires to 1.0.1.
Fixes an issues causing pep8 failures due to a new pep8 release.

(cherry picked from Nova stable)

* Switch to 1000 rounds during unit tests

Fixes bug 992918

passlib 1.6 introduced a minimum number of rounds for sha512_crypt. As
a result, increase the rounds used during testing to the minimum

Change-Id: Ic0c635e92b4f13180a047904a6efa490ab599012
---
 tests/test_overrides.conf |    2 +-
 tools/test-requires       |    2 +-
 tox.ini                   |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/test_overrides.conf b/tests/test_overrides.conf
index 20bc30e..4a05b95 100644
--- a/tests/test_overrides.conf
+++ b/tests/test_overrides.conf
@@ -1,5 +1,5 @@
 [DEFAULT]
-crypt_strength = 10
+crypt_strength = 1000
 
 [identity]
 driver = keystone.identity.backends.kvs.Identity
diff --git a/tools/test-requires b/tools/test-requires
index f2e3c22..43f3d7b 100644
--- a/tools/test-requires
+++ b/tools/test-requires
@@ -11,7 +11,7 @@ nose # for test discovery and console feedback
 nosexcover
 openstack.nose_plugin
 pylint # static code analysis
-pep8==0.6.1 # checks for PEP8 code style compliance
+pep8==1.0.1 # checks for PEP8 code style compliance
 Sphinx>=1.1.2 # required to build documentation
 unittest2 # backport of unittest lib in python 2.7
 webtest # test wsgi apps without starting an http server
diff --git a/tox.ini b/tox.ini
index 1536bf1..a1137b6 100644
--- a/tox.ini
+++ b/tox.ini
@@ -8,7 +8,7 @@ deps = -r{toxinidir}/tools/pip-requires
 commands = bash run_tests.sh -N --no-pep8
 
 [testenv:pep8]
-deps = pep8
+deps = pep8==1.0.1
 commands = pep8 --exclude=vcsversion.py,*.pyc --repeat --show-source bin keystone setup.py
 
 [testenv:hudson]
